### DISCUSSION TITLE
Can we clear force/torque?

### DISCUSSION THREAD

**Discussion by keiohta** [haosulab/SAPIEN#58] [Q&A] [answered]
Hi authors, let me ask one more question related to https://github.com/haosulab/SAPIEN/discussions/56. Would it be possible to reset the force after applying only one step? Like `actor.clear_force` method in the following pseudo code:
```
actor.add_force_at_point()
scene.step()
actor.clear_force()
```
Thanks!

**Comment by fbxiang** [comment, marked as answer]
Force is only good for one step. So you need to keep adding force if you do not want to clear them after each step.

**Reply by keiohta** [reply]
Thanks for the quick reply, which perfectly clarifies my understanding. Thanks!
